### DISCUSSION TITLE
bar cursor disappears in insert mode

### DISCUSSION THREAD

**Discussion by ghost** [helix-editor/helix#10437] [Troubleshooting]
I downloaded this:

<https://github.com/helix-editor/helix/releases/download/22.12/helix-22.12-x86_64-windows.zip>

then set config:

~~~toml
[editor.cursor-shape]
insert = "bar"
~~~

If I enter insert mode, then hold down the Enter key, the cursor disappears until I release. I am currently using GVIM, and I noticed with GVIM, the bar cursor seems to be thicker (perhaps 2px instead of 1px or something), and with GVIM, the cursor seems to change to "always on" or something when holding down Enter key, so that its easy to keep track of it. I am not sure the technology used with both programs, but it just seems the Helix bar cursor is much harder to follow based on the above info.

**Comment by YoRHa-A5** [comment]
Same for me, using 22.08 via chocolatey and the example config:

```toml
[editor.cursor-shape]
insert = "bar"
normal = "block"
select = "underline"
```

Everything but block results in the cursor not appearing at all.

Edit: Took the opportunity to update to 22.12 via scoop instead of choco, but problem persists.
